lib/s3_writer.rb

```diff
@@ -9,6 +9,16 @@ def s3_client
     @s3_client
   end
 
+  def get_author(authorString)
+    checkout_author_info_array = authorString.to_s.empty? ? [] : authorString.to_s.split(",")
```

Comment:
Do you really need the ternary here? I think an empty string split on "," will already give []

Reply:
yes it will. Then we can skip the check here.